### ISSUE TITLE
[Question] The bidirectional cycle-consistency losses advantage compared to pix2pix?

### ISSUE THREAD

**Issue by lxj616** [junyanz/BicycleGAN#12]
Hi, I am really shocked by the awesome work you guys achieved, but yet I get confused about "How BicycleGAN's result differs from pix2pix's result and cyclegan's"

1. Does BicycleGAN give better result at edge2shoes than pix2pix with same training sets and epochs ?
2. Does BicycleGAN trains faster using bidirectional cycle-consistency losses ?
3. In what condition BicycleGAN is better than ... others , and when does not ?

**Comment by junyanz** [comment]
Thanks for your interest in our work. The main goal is to produce multiple outputs given the same input while pix2pix and Cyclegan can only produce 1 result. For your questions: 
1. The performance is similar. 
2. It uses bidirectional loss between the latent space and the output image space.  The training is slightly slower pix2pix as it needs to discover multiple modes.  
3. BicycleGAN can produce multiple results given the same input.

**Comment by lxj616** [comment]
Thank you for your reply, all questions are clear, issue closed : )
